src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
The message in the `doWork` method is not informative enough. It should indicate what type of `Machine` is working. Consider specifying that a `Truck` is working, for example: 'Truck is doing its work.'